casr/src/bin/casr-cluster.rs

```diff
@@ -77,10 +77,7 @@ fn make_clusters(
         (0..len).into_par_iter().for_each(|i| {
             if let Ok(report) = util::report_from_file(casreps[i].as_path()) {
                 if let Ok(trace) = report.filtered_stacktrace() {
-                    let mut crashline = String::new();
-                    if dedup {
-                        crashline = report.crashline;
-                    }
+                    let crashline = report.crashline;
```

Comment:
this var is used just once, you may emplace it

Reply:
Done